.github/workflows/format.yml

```diff
@@ -1,26 +1,26 @@
-name: Formatting check
-on:
-- pull_request
-
-jobs:
-  black:
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.4.1
-        with:
-          access_token: ${{ github.token }}
-
-      - name: Set up Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-
-      - name: Install dependencies
-        run: pip install black
-
-      - uses: actions/checkout@v2
-
-      - name: Run Black
-        run: black -l 100 pennylane_rigetti/ --check

```

Comment:
I don't know what is going on with github, but it is very difficult to review the PR when it says the whole file was changed 😅

Reply:
Apologies! There were DOS (aka CRLF) newlines in some (but not all) of these files, which my editor turned into just “\n”. No substantive changes in most, except for `quilc` versioning and `pip` commands.